autogpt/config/config.py

```diff
@@ -34,7 +34,7 @@ def __init__(self) -> None:
         self.browse_chunk_max_length = int(os.getenv("BROWSE_CHUNK_MAX_LENGTH", 8192))
 
         self.openai_api_key = os.getenv("OPENAI_API_KEY")
-        self.temperature = float(os.getenv("TEMPERATURE", "1"))
+        self.temperature = float(os.getenv("TEMPERATURE", "0"))
```

Comment:
Is this intentional?

Reply:
The effective default was already `0`, since that value is currently in `.env.template` and not commented. I'm just consolidating.